### ISSUE TITLE
训练全部完成，但是weights目录下没有生成pth，日志如下：

### ISSUE THREAD

**Issue by 1006076811** [RVC-Project/Retrieval-based-Voice-Conversion-WebUI#481]
Gradio输出如下：
【
step1:正在处理数据
python trainset_preprocess_pipeline_print.py /home/ducheng/PycharmProjects/Retrieval-based-Voice-Conversion-WebUI/train_data 40000 48 /home/ducheng/PycharmProjects/Retrieval-based-Voice-Conversion-WebUI/logs/tts_test False
step2a:正在提取音高
python extract_f0_print.py /home/ducheng/PycharmProjects/Retrieval-based-Voice-Conversion-WebUI/logs/tts_test 48 harvest
step2b:正在提取特征
python extract_feature_print.py cuda:0 4 0 0 /home/ducheng/PycharmProjects/Retrieval-based-Voice-Conversion-WebUI/logs/tts_test v1
python extract_feature_print.py cuda:0 4 1 1 /home/ducheng/PycharmProjects/Retrieval-based-Voice-Conversion-WebUI/logs/tts_test v1
python extract_feature_print.py cuda:0 4 2 2 /home/ducheng/PycharmProjects/Retrieval-based-Voice-Conversion-WebUI/logs/tts_test v1
python extract_feature_print.py cuda:0 4 3 3 /home/ducheng/PycharmProjects/Retrieval-based-Voice-Conversion-WebUI/logs/tts_test v1
step3a:正在训练模型
write filelist done
python train_nsf_sim_cache_sid_load_pretrain.py -e tts_test -sr 40k -f0 1 -bs 12 -g 0-1-2-3 -te 20 -se 1 -pg pretrained/f0G40k.pth -pd pretrained/f0D40k.pth -l 0 -c 0 -sw 0 -v v1
训练结束, 您可查看控制台训练日志或实验文件夹下的train.log
(128563, 256),3296
training index
adding index
成功构建索引, added_IVF3296_Flat_nprobe_1_tts_test_v1.index
全流程结束！
】

train.log日志如下：
【
2023-06-08 17:08:40,532	tts_test	INFO	{'train': {'log_interval': 200, 'seed': 1234, 'epochs': 20000, 'learning_rate': 0.0001, 'betas': [0.8, 0.99], 'eps': 1e-09, 'batch_size': 12, 'fp16_run': True, 'lr_decay': 0.999875, 'segment_size': 12800, 'init_lr_ratio': 1, 'warmup_epochs': 0, 'c_mel': 45, 'c_kl': 1.0}, 'data': {'max_wav_value': 32768.0, 'sampling_rate': 40000, 'filter_length': 2048, 'hop_length': 400, 'win_length': 2048, 'n_mel_channels': 125, 'mel_fmin': 0.0, 'mel_fmax': None, 'training_files': './logs/tts_test/filelist.txt'}, 'model': {'inter_channels': 192, 'hidden_channels': 192, 'filter_channels': 768, 'n_heads': 2, 'n_layers': 6, 'kernel_size': 3, 'p_dropout': 0, 'resblock': '1', 'resblock_kernel_sizes': [3, 7, 11], 'resblock_dilation_sizes': [[1, 3, 5], [1, 3, 5], [1, 3, 5]], 'upsample_rates': [10, 10, 2, 2], 'upsample_initial_channel': 512, 'upsample_kernel_sizes': [16, 16, 4, 4], 'use_spectral_norm': False, 'gin_channels': 256, 'spk_embed_dim': 109}, 'model_dir': './logs/tts_test', 'experiment_dir': './logs/tts_test', 'save_every_epoch': 1, 'name': 'tts_test', 'total_epoch': 20, 'pretrainG': 'pretrained/f0G40k.pth', 'pretrainD': 'pretrained/f0D40k.pth', 'version': 'v1', 'gpus': '0-1-2-3', 'sample_rate': '40k', 'if_f0': 1, 'if_latest': 0, 'save_every_weights': '0', 'if_cache_data_in_gpu': 0}
2023-06-08 17:08:46,036	tts_test	INFO	loaded pretrained pretrained/f0G40k.pth pretrained/f0D40k.pth
】

**Comment by 1006076811** [comment]
原来是cuda 内存爆了
